extensions/ql-vscode/src/databaseFetcher.ts

```diff
@@ -3,8 +3,7 @@ import * as unzipper from 'unzipper';
 import { zip } from 'zip-a-folder';
 import {
   Uri,
-  ProgressOptions,
-  ProgressLocation,
+  CancellationToken,
```

Comment:
This file is mostly about disentangling all the error handling and progress reporting.